popup/components/sections/Features.js

```diff
@@ -1,9 +1,27 @@
 import Section from "../layout/Section";
 import SwitchControl from "../controls/SwitchControl";
+import { RiQuestionLine } from "@remixicon/react";
+import { Tooltip } from "react-tooltip";
 
 const Layout = () => (
     <Section title="Funcionalidades">
-        <SwitchControl label="Relembrar utilizador" storageKey="autoLogin" />
+        <SwitchControl
+            label={
+                <>
+                    Relembrar utilizador
+                    <span className="question-icon">
+                        <RiQuestionLine data-tooltip-id="autoLogin-tooltip" />
+                    </span>
+                    <Tooltip 
+                        className="tooltip"
+                        id="autoLogin-tooltip" 
+                        content={ <> Depois de ativada a opção, <br />o login deve ser feito pelas credenciais <br /> não pela autenticação federada. </>} 
+                        place="top"
+                    />
+                </>
+            }
```

Comment:
Since this is now supposed to be used in all switches, it should be moved to the switch itself. In this case, `label` would be "Relembrar utilizador" and the `content` within the tooltip should also be passed as a prop.